checkpoint-single-nic/azuredeploy.json

```diff
@@ -0,0 +1,305 @@
+                        }
+                    ]
+                }
+            }
+        },
+        "plan": "[variables(concat('plan', parameters('offer')))]",
+        "planR7730BYOL": {
+            "name": "sg-byol",
+            "product": "[variables('imageOffer')]",
+            "publisher": "[variables('imagePublisher')]"
+        },
+        "planR7730PAYG": {
+            "name": "sg-ngtp",
+            "product": "[variables('imageOffer')]",
+            "publisher": "[variables('imagePublisher')]"
+        },
+        "publicIPAddressName": "security-gateway",
+        "publicIPAddressId": "[resourceId('Microsoft.Network/publicIPAddresses', variables('publicIPAddressName'))]",
+        "storageAccountType": "Standard_LRS",
+        "subnet1Name": "external-subnet",
+        "subnet1Prefix": "10.0.1.0/24",
+        "subnet1Id": "[concat(variables('vnetId'),'/subnets/',variables('subnet1Name'))]",
+        "subnet2Name": "internal-subnet",
+        "subnet2Prefix": "10.0.2.0/24",
+        "subnet2Id": "[concat(variables('vnetId'),'/subnets/',variables('subnet2Name'))]",
+        "vnetId": "[resourceId('Microsoft.Network/virtualNetworks',variables('vnetName'))]",
+        "vnetName": "vnet",
+
+        "internalRouteTableName": "InternalRouteTable",
+        "internalRouteTableId": "[resourceId('Microsoft.Network/routeTables', variables('internalRouteTableName'))]"
+    },
+    "resources": [
+        {
+            "apiVersion": "[variables('apiVersion')]",
+            "location": "[resourceGroup().location]",
+            "name": "[variables('internalRouteTableName')]",
+            "properties": {
+                "routes": [
+                    {
+                        "name": "Default",
+                        "properties": {
+                            "addressPrefix": "0.0.0.0/0",
+                            "nextHopType": "VirtualAppliance",
+                            "nextHopIpAddress": "[variables('gatewayPrivateAddress')]"
+                        }
+                    },
+                    {
+                        "name": "On-Premise-Network",
+                        "properties": {
+                            "addressPrefix": "[variables('onPremiseNetwork')]",
+                            "nextHopType": "VirtualAppliance",
+                            "nextHopIpAddress": "[variables('gatewayPrivateAddress')]"
+                        }
+                    },
+                    {
+                        "name": "External-Subnet",
+                        "properties": {
+                            "addressPrefix": "[variables('subnet1Prefix')]",
+                            "nextHopType": "VirtualAppliance",
+                            "nextHopIpAddress": "[variables('gatewayPrivateAddress')]"
+                        }
+                    }
+                ]
+            },
+            "type": "Microsoft.Network/routeTables"
```

Comment:
Done.

From: Kay Singh [mailto:notifications@github.com]
Sent: Tuesday, December 29, 2015 8:46 PM
To: Azure/azure-quickstart-templates
Cc: Tamir Zegman
Subject: Re: [azure-quickstart-templates] Use Static public addresses, allocate additional public addresses to simulate web apps. (#1033)


In checkpoint-single-nic/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/1033#discussion_r48560686>:

> +                            "addressPrefix": "[variables('onPremiseNetwork')]",

> +                            "nextHopType": "VirtualAppliance",

> +                            "nextHopIpAddress": "[variables('gatewayPrivateAddress')]"

> +                        }

> +                    },

> +                    {

> +                        "name": "External-Subnet",

> +                        "properties": {

> +                            "addressPrefix": "[variables('subnet1Prefix')]",

> +                            "nextHopType": "VirtualAppliance",

> +                            "nextHopIpAddress": "[variables('gatewayPrivateAddress')]"

> +                        }

> +                    }

> +                ]

> +            },

> +            "type": "Microsoft.Network/routeTables"

Can you put the resource types at the top? It's hard to identify the resources with the types all the way at the bottom

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/1033/files#r48560686>.


Email secured by Check Point.